backend/api/Startup.cs

```diff
@@ -236,15 +237,15 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, IApiVers
 
             app.UseSwagger(options =>
             {
-                options.RouteTemplate = "/swagger/{documentname}/swagger.json";
+                options.RouteTemplate = "/api-docs/swagger/{documentname}/swagger.json";
```

Comment:
The original route doesn't appear to work on OpenShift.  Trying to see if a different route will work.

Reply:
Please change to `"/api/swagger/{documentname}/swagger.json"`

We don't have a reverse proxy rule for "api-docs" (yet) but we do proxy everything for `/api`

Reply:
`/api/swagger/...` doesn't work.  I've tested it already.  I think I'm going to instead create config vars for all these settings.

Reply:
Sure. Let me know when you find something that fixes it and I will re review